src/ui/components/Badge/styles.scss

```diff
@@ -0,0 +1,35 @@
+@import "~core/css/inc/mixins";
+@import "~ui/css/vars";
+
+@mixin icon($name) {
+  background: url("~amo/img/icons/badge-#{$name}.svg") 50% 50% no-repeat;
```

Comment:
I would rather keep the badges:

1. in the UI folder
2. Inside the badge component folder

We (mostly) do this with `Icon` and I think it's a bit easier to manage.